comps/cores/mega/orchestrator.py

```diff
@@ -208,6 +211,7 @@ async def execute(
                 hitted_ends = [".", "?", "!", "。", "，", "！"]
                 downstream_endpoint = self.services[downstream[0]].endpoint_path
 
+            @opea_telemetry
             def generate():
```

Comment:
This is case 2) so simply add @opea_telemetry does not produce correct duration for `generate` here.